### ISSUE TITLE
Migrating from Libreboard

### ISSUE THREAD

**Issue by lepalmtree** [wekan/wekan#502]
Hi,
I'm currently using a Libreboard 0.6 instance on my own server. I would like to migrate to Wekan. Is there any way I could keep the board I already have on Libreboard ?

**Comment by b-long** [comment]
:+1: It might be useful to make this an addendum to the wiki : https://github.com/wekan/wekan/wiki/Install-and-Update

**Comment by mquandalle** [comment]
How did you install Libreboard v0.6? Upgrade should be as simple as stopping the old version and starting the new version, data migration to the new format will happen automatically on the new version start.

**Comment by lepalmtree** [comment]
Well I git cloned it. I tried to start wekan but I couldn't log in. It's
like the user doesn't exist. My libreboard 0.6 still work though
Le 30 janv. 2016 12:07, "Maxime Quandalle" <notifications@github.com> a
écrit :

**Comment by mquandalle** [comment]
Did you use the `meteor` command to start Wekan?

**Comment by lepalmtree** [comment]
Yes I did, just like I do for libreboard
Le 30 janv. 2016 12:36, "Maxime Quandalle" <notifications@github.com> a
écrit :

**Comment by mquandalle** [comment]
In this case the database is located under a local directory (`.meteor/local`) and so if you clone the new version in a separate directory you will start from a blank database. You should either copy paste the data from the old directory to the new one, or another solution is to `git pull` the new version directly from your original directory.

**Comment by lepalmtree** [comment]
Alright, I copied
libreboard/.meteor/local/db to wekan/.meteor/local/db and it didn't work
so I tried
libreboard/.meteor/local to wekan/.meteor/local
and it doesn't work either. It tells me "Authentification interdite" 
(French version) so "Forbidden authentication" I guess in english.
I also tried to git pull from my libreboard directory, from the master 
branch and it tells me :
"fatal: repository 
'https://github.com/wekan/wekan/libreboard/libreboard.git/" which is 
normal since the github repo got closed.
Did I do something wrong ?
